py/objint_longlong.c

```diff
@@ -40,14 +40,6 @@
 
 #if MICROPY_LONGINT_IMPL == MICROPY_LONGINT_IMPL_LONGLONG
 
-// Python3 no longer has "l" suffix for long ints. We allow to use it
```

Comment:
Well, commit message has a surface bug ("stale" duplicated) and ungrounded assumption of "hasn't been used in years". We just recently found yet another bug in long int handling, and this diagnostic measure is there exactly to allow to debug such cases easier. So I'm not sure if it "hasn't been used in years", but everyone certainly should be using it every time needed and it shouldn't be removed, certainly not as part of unrelated changes.